android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -1794,7 +1801,7 @@ private TabImpl getTabImpl() {
         return (TabImpl) getTab();
     }
 
-    @Override
+    /*@Override
```

Comment:
why are we commenting this out?

Reply:
It was WIP commit, eventually it was removed later.